charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -1,6 +1,20 @@
 clusterName: demo
 opentelemetry-operator:
   enabled: true
+defaultCRConfig:
+  targetAllocator:
+    image: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:main
+    prometheusCR:
+      enabled: true
+      podMonitorSelector:
+        matchLabels:
+          app: my-app
+      scrapeInterval: "30s"
+      serviceMonitorSelector:
+        matchExpressions:
+          - key: kubernetes.io/app-name
+            operator: In
+            values: ["my-app"]
```

Comment:
@sergeybataev would you mind moving this to a new example? I'm trying to have this example be something a user would install to get started with otel and a new example would be great for the prometheus specific use case. I'm going to be working on this for #1080